dispatches/models/fossil_case/ultra_supercritical_plant/ultra_supercritical_powerplant.py

```diff
@@ -440,11 +402,52 @@ def constraint_out_pressure(b, t):
     @m.fs.Constraint(m.fs.time)
     def constraint_bfp_power(b, t):
         return (
-            b.booster.control_volume.work[t] + b.bfp.control_volume.work[t]
+            b.booster.control_volume.work[t]
```

Comment:
Minor comment: PEP8 standard puts operators at end of line, rather than beginning of next line.